packages/server/src/cron/delete-old-builds.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
```

Comment:
could you add a license comment to the top of this file? you can copy/paste from the psi-collect one :)